compiler/x/codegen/OMRCodeGenerator.cpp

```diff
@@ -3062,21 +3060,9 @@ void OMR::X86::CodeGenerator::dumpPostGPRegisterAssignment(TR::Instruction * ins
    }
 #endif
 
-bool
-OMR::X86::CodeGenerator::useSSEFor(TR::DataType type)
-   {
-   if (type == TR::Float)
-      return self()->useSSEForSinglePrecision();
-   else if (type == TR::Double)
-      return self()->useSSEForDoublePrecision();
-   else
-      return false;
-   }
-
 bool
 OMR::X86::CodeGenerator::needToAvoidCommoningInGRA()
```

Comment:
This function is not needed anymore. There is a default implementation that returns false in /codegen/OMRCodeGenerator.hpp. Also, it is not implemented on any of the other architectures and is only called in optimizer/GlobalRegisterAllocator.cpp. Perhaps that code can be folded as well.

Reply:
Yeah, I debated whether to leave in the option to avoid commoning and decided to keep it.  However, in thinking about it some more it really was an x87 thing so I'm fine with just removing it.  Thanks for pointing it out.

Reply:
Removed in dbb30e37e67cb68d794227dcb012c41ef810ede9